File: src/main/java/com/rs/java/game/area/multi/ApeAtollMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class ApeAtollMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "White Wolf Mountain Multi";
	}

	public ApeAtollMulti() {
		super(10794, 10795, 11051, 11050);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/ApeAtollMulti.kt
package com.rs.kotlin.game.world.area.areas.multi
import com.rs.kotlin.game.world.area.Area
class ApeAtollMulti : Area(10794, 10795, 11051, 11050) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "White Wolf Mountain Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
